configurations/minimal.yaml

```diff
@@ -1,97 +1,61 @@
-# File metadata, to be printed by the simulator
-name:
+name: # File metadata, to be printed by the simulator
   "Minimal Dispatch Example"
 
-# Defines the bounds for case and ambulance random location initialization
-location_generator:
-  class: CircleLocationGenerator
-  classpath: ems.generators.location.circle
+bases: # Initializes 6 bases in the UCSD area
+  class: ems.datasets.location.location_set.LocationSet
+  latitudes: [32.877328, 32.873803, 32.876727, 32.875811, 32.884714, 32.888589]
+  longitudes: [-117.234695, -117.239527, -117.242650, -117.230869, -117.236528, -117.242113]
 
-  # Defines coordinates for the center of the circle
+ambulances: # Initializes ambulances; one stationed at each of the 6 bases
+  class: ems.datasets.ambulance.custom_ambulance_set.CustomAmbulanceSet
+  bases: "@bases"
```

Comment:
Would it make more sense to name the first order object differently from the reference? For example: 

`bases: @bases_set`

Reply:
Another question: Will `@ base_set` work?

Reply:
Hmmm I feel like it would be better to keep the naming the same

Reply:
Well, if we explain `@` clearly, it should be no problem.